tests/base.conf.js

```diff
@@ -40,6 +40,7 @@ class BaseConfig {
 
         browser.driver.wait(setupSettings, 5 * 1000, 'Settings should be setup within 5 seconds');
         browser.driver.wait(setupUser, 5 * 1000, 'User should be setup within 5 seconds');
+        browser.driver.wait(setupUserContact, 5 * 1000, 'User contact should be setup within 5 seconds');
```

Comment:
I think the userContact should be set up before updating the user to refer to it, just to avoid any race conditions.

Reply:
Done